gammagl/models/hardgat.py

```diff
@@ -0,0 +1,50 @@
+import tensorlayerx as tlx
+from gammagl.layers.conv import HardGATConv
+
+
+class HardGATModel(tlx.nn.Module):
+    r"""The graph attentional operator from the `"Graph Attention Networks"
+    <https://arxiv.org/abs/1710.10903>`_ paper
+
+    Parameters
+    ----------
+        feature_dim (int): input feature dimension
+        hidden_dim (int): hidden dimension
+        num_class (int): number of classes
+        heads (int): number of attention heads
+        drop_rate (float): dropout rate
+        name (str): model name
```

Comment:
The Parameter in this doc should correspond to the Parameters in the following ``__init__`` function. You lose the Parameter k.